easybuild/easyconfigs/p/PyTorch-bundle/PyTorch-bundle-2.1.2-foss-2023a-CUDA-12.1.1.eb

```
@@ -0,0 +1,104 @@
+easyblock = 'PythonBundle'
+
+name = 'PyTorch-bundle'
+version = '2.1.2'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://pytorch.org/'
+description = """PyTorch with compatible versions of official Torch extensions."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('RE2', '2023-08-01'),  # for torchtext
+    ('parameterized', '0.9.0'),
```

```suggestion
    ('parameterized', '0.9.0'),
    ('scikit-learn', '1.3.1'),
    ('matplotlib', '3.7.2'),
```
sklearn is used a bit here and there. Matplotlib I think it was in pytorch-ignite: tests/ignite/engine